Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -985,6 +985,10 @@
 #define ENABLE_WEBDRIVER_WHEEL_INTERACTIONS 1
 #endif
 
+#if !defined(ENABLE_WEBDRIVER_BIDI) && (PLATFORM(MAC) || PLATFORM(IOS) || PLATFORM(IOS_SIMULATOR) || PLATFORM(VISION))
```

Comment:
`PLATFORM(IOS_SIMULATOR)` is redundant here, as `PLATFORM(IOS)` already includes the Simulator.